pom.xml

```diff
@@ -584,19 +626,21 @@ https://gitbox.apache.org/repos/asf?p=maven-pmd-plugin.git;a=blob_plain;f=src/ma
             <goals>
               <goal>prepare-agent</goal>
             </goals>
+            <phase>initialize</phase>
```

Comment:
what does this change do?

Reply:
It's redundant, the default is already initialize